crates/sui/src/client_commands.rs

```diff
@@ -188,9 +188,9 @@ pub enum SuiClientCommands {
     #[clap(name = "addresses")]
     Addresses,
 
-    /// Generate new address and keypair.
+    /// Generate new address and keypair, with optional keypair scheme, default to Ed25519.
```

Comment:
lets put the list of options in here as well, so user will know what are the expected input when they use the help command